src/components/Sidebar.jsx

```diff
@@ -0,0 +1,58 @@
+import * as React from "react";
+import Tabs from "@mui/material/Tabs";
+import Tab from "@mui/material/Tab";
+import HomeIcon from "@mui/icons-material/Home";
+import ChatBubbleIcon from "@mui/icons-material/ChatBubble";
+import GroupsIcon from "@mui/icons-material/Groups";
+import LogoutIcon from "@mui/icons-material/Logout";
+import { Box, Typography } from "@mui/material";
+import { sidebarStyle } from "./Sidebar.style";
+import { Link } from "react-router-dom";
+
+export default function Sidebar() {
+  const { classes } = sidebarStyle();
+  const [value, setValue] = React.useState(0);
+
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+  };
+
+  return (
+    <Box className={classes.styleBox}>
+      <Box className={classes.sidebarIconContStyles}>
```

Comment:
This inner component seems to get repeated 3 times below. Use a map function and write is once. Do not repeat your code